windows-build/Dockerfile

```diff
@@ -3,13 +3,15 @@ FROM archlinux/archlinux:base-devel
 ENV MAKEFLAGS -j8
 ENV CGO_ENABLED 1
 
-RUN pacman -Syu --noconfirm &&\
-	pacman -S --noconfirm git go multilib-devel &&\
-	useradd -m builder &&\
-	git clone --depth 1 -b go-cgo-gcc-parallelism-1.20.3 https://github.com/Evengard/go &&\
-	cd go/src && ./make.bash && pacman -R --noconfirm go &&\
-	echo "builder ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/builder &&\
-	su builder -c "cd &&\
+RUN echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist" >> /etc/pacman.conf
```

Comment:
Also, @XandrosDarkstorm do not split `RUN` directives in "production" Docker files. If all bash commands are in a single `RUN`, they end up as a single Docker image layer, which is usually faster to fetch.

Reply:
Yes, i realize the problem. It was done for clarity during debugging the problem. I'll merge the command into one.